coursier.bzl

```diff
@@ -15,7 +15,8 @@ load("//third_party/bazel_json/lib:json_parser.bzl", "json_parse")
 load("//:specs.bzl", "utils")
 load(
     "//:private/versions.bzl",
-    "COURSIER_CLI_MAVEN_PATH",
+    "COURSIER_CLI_GITHUB_ASSET_URL",
+    "COURSIER_CLI_BAZEL_MIRROR_URL",
```

Comment:
nit: does alphabetization not matter here? I'd expect buildifier to fix this (but if it doesn't, no worries)